manifests/hwtools/package.pp

```diff
@@ -17,21 +17,21 @@
       # preventing the proper upgrade. To fix it we need to
       # 'yum erase smbios-utils-python python-smbios'
       package { [
-        'smbios-utils-python',
-        'python-smbios',
-      ]:
-        ensure => purged,
-        before => Package['dell-system-update'],
+          'smbios-utils-python',
```

Comment:
This is relative to formating, not a puppet 8 issue. 
Please open another PR for this .